### PR TITLE
Add cron_helper to report success/failure to cron-daily Slack room

### PR DESCRIPTION
**Pull request by ewjordan**
Also causes secondary backup script to report success/failure.

### CODE REVIEW

File: bin/cron/push_latest_rds_backup_to_secondary_account [hunk 1]
```diff
@@ -4,6 +4,10 @@ Dir.chdir(File.expand_path('../../../dashboard', __FILE__))
 require_relative '../../dashboard/config/environment'
 require 'aws-sdk'
 require 'cdo/chat_client'
+require 'cdo/cron_helper'
+require 'cdo/only_one'
+
+TASK_NAME = 'Backup RDS Snapshot To Secondary Account'
```
Comment (ashercodeorg) [inline bin/cron/push_latest_rds_backup_to_secondary_account]: Use `freeze` on CONSTANTS.

File: bin/cron/push_latest_rds_backup_to_secondary_account [hunk 2]
```diff
@@ -90,6 +94,12 @@ end
 # which requires setting appropriate secrets in our Chef config (which will come through via crontab.erb)
 def main
   ChatClient.message 'server operations', 'Beginning cross-account RDS backup'
+
+  unless only_one_running?(__FILE__)
+    CronHelper.report_cron_daily_result TASK_NAME, false
+    ChatClient.log 'Cross-account RDS backup failed, previous task is still running!', color: 'red', notify: 1
```
Comment (ashercodeorg) [inline bin/cron/push_latest_rds_backup_to_secondary_account]: What does `1` mean for `notify`? The same as `true`?

File: lib/cdo/cron_helper.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'cdo/chat_client'
+
+class CronHelper
+  CRON_REPORT_ROOM = 'cron-daily'.freeze
+  def self.report_cron_daily_result(task_name, success)
```
Comment (ashercodeorg) [inline lib/cdo/cron_helper.rb]: nit: Blank line above new method.

File: lib/cdo/cron_helper.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    status = success ? 'success' : 'failure'
+    result_string = "#{task_name} : #{status}"
```
Comment (ashercodeorg) [inline lib/cdo/cron_helper.rb]: super-nit: Remove space before colon.

**Review by ashercodeorg — CHANGES_REQUESTED** [review]
At a high level, I'll continue to voice my disagreement with this approach. Rather than depend on people to monitor rooms for positives, we should be alerting on negatives.

At the very least, any time there is a red message, I should be seeing a `Honeybadger.notify`.

File: bin/cron/push_latest_rds_backup_to_secondary_account (hunk 2 above)
Comment (ewjordan) [reply bin/cron/push_latest_rds_backup_to_secondary_account]: I think that was a HipChat holdover - removing.

**Pull request closed without merging**